packages/resourceful-redux/test/unit/reducers/delete.js

```diff
@@ -41,6 +42,7 @@ describe('reducers: delete', function() {
             3: {id: 3},
             4: {id: 4},
           },
+          lists: {},
```

Comment:
these tests should utilize a skeleton from something like `utils/generate-default-initial-state` so we don't gotta update them any time a new major key is added (or removed)

Reply:
Yup, I think we chatted about this in an earlier comment. This isn't a blocker, and should be done in a follow up PR.